testcases/PowerNVDump.py

```diff
@@ -933,30 +934,53 @@ def runTest(self):
             self.setup_test()
             self.c.run_command("ppc64_cpu --smt=%s" % i, timeout=180)
             self.c.run_command("ppc64_cpu --smt")
-            log.debug("=============== Testing kdump/fadump with smt=%s ===============" % i)
+            log.info("=============== Testing kdump/fadump with smt=%s ===============" % i)
             boot_type = self.kernel_crash()
             self.verify_dump_file(boot_type)
         self.setup_test()
         self.c.run_command("ppc64_cpu --cores-on=1", timeout=180)
         self.c.run_command("ppc64_cpu --cores-on")
-        log.debug("=============== Testing kdump/fadump with single core ===============")
+        log.info("=============== Testing kdump/fadump with single core ===============")
         boot_type = self.kernel_crash()
         self.verify_dump_file(boot_type)
         self.setup_test()
         self.c.run_command("ppc64_cpu --cores-on=1", timeout=180)
         self.c.run_command("ppc64_cpu --smt=off", timeout=180)
-        log.debug("=============== Testing kdump/fadump with single cpu ===============")
+        log.info("=============== Testing kdump/fadump with single cpu ===============")
         boot_type = self.kernel_crash()
         self.verify_dump_file(boot_type)
         if self.is_lpar:
             for i in ["off", "2", "4", "on"]:
                 self.setup_test()
                 self.c.run_command("ppc64_cpu --smt=%s" % i, timeout=180)
                 self.c.run_command("ppc64_cpu --smt")
-                log.debug("=============== Testing kdump/fadump with smt=%s and dumprestart from HMC ===============" % i)
+                log.info("=============== Testing kdump/fadump with smt=%s and dumprestart from HMC ===============" % i)
                 boot_type = self.kernel_crash(crash_type="hmc")
                 self.verify_dump_file(boot_type)
 
+class KernelCrash_KdumpDLPAR(PowerNVDump, testcases.OpTestDlpar.OpTestDlpar):
+
+    # This test verifies kdump/fadump after cpu and memory add/remove.
+    # cpu_resource and mem_resource must be defined in ~/.op-test-framework.conf.
+    # cpu_resource - max number of CPU
+    # mem_resource - max memory in MB
+    # Ex: cpu_resource=4
+    #     mem_resource=2048
+
+    def runTest(self):
+        self.extended = {'loop':0,'wkld':0,'smt':8}
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
+        for component in ['proc', 'mem']:
+            for operation in ['a', 'r']:
+                self.setup_test()
+                if component == "proc":
+                    self.AddRemove("proc", "--procs", operation, self.cpu_resource)
```

Comment:
can we do single operation like add cpu and trigger kdump and next one have remove cpu and trigger kdump

Reply:
yeah this testcase will be run 4 times with [ proc , a ] [ proc, r ] [ mem, a ] [ mem, r ]. Raising new PR with logs.